Server/controllers/authController.js

```
@@ -2,458 +1,371 @@
-      .buildAndSendEmail(
-        "welcomeEmailTemplate",
-        { name: newUser.firstName },
-        newUser.email,
-        "Welcome to Uptime Monitor"
-      )
-      .catch((error) => {
-        logger.error("Error sending welcome email", {
-          service: SERVICE_NAME,
-          error: error.message,
+        const userForToken = {...newUser._doc};
+        delete userForToken.profileImage;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)